src/shogun/machine/LinearMachine.cpp

```diff
@@ -120,3 +141,50 @@ void CLinearMachine::store_model_features()
 {
 }
 
+float64_t CLinearMachine::compute_bias(CFeatures* data)
+{
+	REQUIRE(m_labels,"No labels set\n");
+
+	if (!data)
+		data=features;
+
+	REQUIRE(data,"No features set\n");
+
+	REQUIRE(m_labels->get_num_labels() == data->get_num_vectors(),"Number of training vectors does not match number of labels\n");
+
+	SGVector<float64_t> outputs = apply_get_outputs(data);
+
+	int32_t num_vec=data->get_num_vectors();
+
+	Map<VectorXd> eigen_outputs(outputs,num_vec);
+	Map<VectorXd> eigen_labels(((CRegressionLabels*)m_labels)->get_labels(),num_vec);
+
+	return (eigen_labels - eigen_outputs).mean() ;
```

Comment:
BTW I think linalg has mean computation in it. Could you check that and use if the case.
If not, can you open an issue that is to implement mean computations in linalg?